core/modules/node/node.install

```diff
@@ -1119,6 +1119,17 @@ function node_update_1013() {
   cache_clear_all();
 }
 
+/**
```

Comment:
@quicksketch does this update hook belong inside or outside of the `"addtogroup updates-7.x-to-1.x"`?